shared/helpers/view.js

```diff
@@ -14,6 +14,16 @@ module.exports = function (Handlebars) {
     viewOptions = options.hash || {};
     app = getProperty('_app', this, options);
 
+    _.forEach(_.omit(viewOptions, 'templateName'), function(value, key) {
+      if (
+        key.match(/[A-Z]/g) &&
+        !app.modelUtils.isModel(value) &&
+        !app.modelUtils.isCollection(value)
+      ) {
+        throw new Error('Template properties should be passed in snake_case: "' + key + '"')
```

Comment:
doesn't _have_ to be snake_cased - but basically since it's just HTML it's case insensitive. so you can probably do something like `myAttr` and then `attr = myattr || myAttr`